src/client/phy-ue.cc

```diff
@@ -438,11 +435,25 @@ void PhyUe::Start() {
                 fft_dlpilot_counters_.CompleteSymbol(frame_id);
             if (pilot_fft_complete) {
               if (kPrintPhyStats) {
-                this->phy_stats_->PrintDlSnrStats(frame_id, ant_id);
+                this->phy_stats_->PrintDlSnrStats(frame_id);
               }
               if (kEnableCsvLog) {
-                this->phy_stats_->RecordDlPilotSnr(
-                    csv_loggers_.at(CsvLog::kDLPSNR).get(), frame_id, ant_id);
+                constexpr size_t kRecScNum = 3;
+                //set subcarriers to record DL CSI
+                constexpr std::array<size_t, kRecScNum> csi_rec_sc = {20, 140,
```

Comment:
All values like 20,140, 200 should have names.  it is not obvious why you picked these values.